pkg/skaffold/docker/dependencies.go

```diff
@@ -260,10 +259,17 @@ func WalkWorkspace(workspace string, excludes, deps []string) (map[string]bool,
 		absFrom := filepath.Join(workspace, dep)
 
 		keepFile := func(path string, info walk.Dirent) (bool, error) {
+			// This region was never reached
 			// Always keep root folders.
-			if info.IsDir() && path == absFrom {
-				return true, nil
+			// if info.IsDir() && path == absFrom {
+			// 	return false, nil
+			// }
+
```

Comment:
Remove this code (I'm not sure what line 262 is referring to)

Reply:
I wanted to show  (for discussion) that this `if` statement always evaluates to false since the `keepfile` function was only being called for files and not directories. Also, if we add the root directory tests fail so I initially set it to false but then I realized we don't need that part at all (hence commented out and ready to be removed if there's an agreement)

Reply:
Ah makes sense 👍

Reply:
These comments need to be cleaned up.